docs/docs/classic-ml/webhooks/index.mdx

```diff
@@ -468,15 +468,14 @@ if __name__ == "__main__":
 
    ```python
    from mlflow import MlflowClient
-   from mlflow.entities.webhook import WebhookAction, WebhookEntity, WebhookEvent
 
    client = MlflowClient("http://localhost:5000")
 
    # Create webhook with HMAC verification
    webhook = client.create_webhook(
        name="fastapi-receiver",
        url="https://your-domain.com/webhook",
-       events=[WebhookEvent(WebhookEntity.MODEL_VERSION, WebhookAction.CREATED)],
```

Comment:
This is valid code, but it feels more complex than simply using `"model_version.created"`.

Reply:
You don't need to remember. The IDE tells you what options are available:

<img width="722" height="157" alt="image" src="https://github.com/user-attachments/assets/2e6dfe27-6a59-4921-aa46-8fdc5c36cf55" />

Reply:
but this only works for IDEs like vscode.